source/faq/faq.rst

```diff
@@ -203,7 +204,7 @@ device of type MQTT with two attributes (temperature and humidity). Then you
 set your actual device to push the data to dojot.
 
 .. ATTENTION::
-  As of **v0.5.2**, you can choose the between two MQTT brokers: Mosca or VerneMQ. By default,
+  As of **v0.6.0**, you can choose the between two MQTT brokers: Mosca or VerneMQ. By default,
```

Comment:
I believe that this snippet does not change, but perhaps the sentence could begin with: 
> Since version 0.5.2, you can choose the between two [...]